observation_portal/settings.py

```diff
@@ -187,21 +187,21 @@
 # https://docs.djangoproject.com/en/2.1/howto/static-files/
 
 AWS_STORAGE_BUCKET_NAME = os.getenv('AWS_BUCKET_NAME', 'observation-portal-test-bucket')
-AWS_REGION = os.getenv('AWS_REGION', 'us-west-2')
+AWS_S3_REGION_NAME = os.getenv('AWS_REGION', 'us-west-2')
 AWS_ACCESS_KEY_ID = os.getenv('AWS_ACCESS_KEY_ID', None)
 AWS_SECRET_ACCESS_KEY = os.getenv('AWS_SECRET_ACCESS_KEY', None)
-AWS_S3_CUSTOM_DOMAIN = 's3-us-west-2.amazonaws.com/{}'.format(AWS_STORAGE_BUCKET_NAME)
```

Comment:
yeah I'm not sure why we had this... maybe just our lack of knowledge back then. If it works with uploading and accessing older uploaded files, then that's fine